File: FinalProject/TrafficOffenceManagementSystem/TrafficOffenceManagementSystemFrontend/src/app/model/fine-response.ts
import { Fine } from '../model/fine';

export class FineResponse {

    statusCode: number;
    message: string;
   // data: Fine[];
    data: Array<Fine> = [];

    constructor(statusCode: number, message: string, data: Array<Fine>){

        this.statusCode = statusCode;
        this.message = message;
        this.data = data;
    }
}

File: FinalProject/TrafficOffenceManagementSystem/offence-service/src/main/java/com/janitha/trafficoffencemanagement/offenceservice/dto/FineResponse.java
package com.janitha.trafficoffencemanagement.offenceservice.dto;

import com.janitha.trafficoffencemanagement.model.offenceservice.Fine;

import java.util.List;

public class FineResponse {

    private int statusCode;
    private String message;
    private Object data;

    public int getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(int statusCode) {
        this.statusCode = statusCode;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Object getData() {
        return data;
    }

    public void setData(Object data) {
        this.data = data;
    }
}
